mods/d2k/tilesets/arrakis.yaml

```diff
@@ -6731,3 +6731,1537 @@ Templates:
+			Template: 600
+			Segment:
+				Start: RockRockCliff.L
+				Inner: RockRockCliff
+				End: RockRockCliff.D
+				Points: 2,1, 1,1, 1,2
+		MultiBrush@179:
+			Template: 613
+			Segment:
+				Start: RockRockCliff.L
+				Inner: RockRockCliff
+				End: RockRockCliff.U
+				Points: 2,1, 1,1, 1,0
+		MultiBrush@180:
+			Template: 614
+			Segment:
+				Start: RockRockCliff.D
+				Inner: RockRockCliff
+				End: RockRockCliff.L
+				Points: 1,0, 1,1, 0,1
+		MultiBrush@181:
+			Template: 615
+			Segment:
+				Start: RockRockCliff.R
+				Inner: RockRockCliff
+				End: RockRockCliff.D
+				Points: 0,1, 1,1, 1,2
+		MultiBrush@182:
+			Template: 616
+			Segment:
+				Start: RockRockCliff.U
+				Inner: RockRockCliff
+				End: RockRockCliff.R
+				Points: 1,2, 1,1, 2,1
+		MultiBrush@183: # ROCK ROCK END
+			Template: 610
+			Segment:
+				Start: RockRockCliff.D
+				Inner: RockRockCliff
+				End: RockRockCliffEnd.D
+				Points: 1,0, 1,1, 1,2
+		MultiBrush@184:
+			Template: 611
+			Segment:
+				Start: RockRockCliffEnd.D
+				Inner: RockRockCliff
+				End: RockRockCliff.D
+				Points: 1,0, 1,1
+		MultiBrush@185:
+			Template: 635
+			Segment:
+				Start: RockRockCliffEnd.U
+				Inner: RockRockCliff
+				End: RockRockCliff.U
+				Points: 1,2, 1,1, 1,0
+		MultiBrush@186:
+			Template: 640
+			Segment:
+				Start: RockRockCliff.U
+				Inner: RockRockCliff
+				End: RockRockCliffEnd.U
+				Points: 1,2, 1,1, 1,0
+		MultiBrush@187:
+			Template: 636
+			Segment:
+				Start: RockRockCliffEnd.R
+				Inner: RockRockCliff
+				End: RockRockCliff.R
+				Points: 0,1, 1,1, 2,1
+		MultiBrush@188:
+			Template: 639
+			Segment:
+				Start: RockRockCliff.R
+				Inner: RockRockCliff
+				End: RockRockCliffEnd.R
+				Points: 0,1, 1,1, 2,1
+		MultiBrush@189:
+			Template: 637
+			Segment:
+				Start: RockRockCliff.L
+				Inner: RockRockCliff
+				End: RockRockCliffEnd.L
+				Points: 2,1, 1,1, 0,1
+		MultiBrush@190:
+			Template: 638
+			Segment:
+				Start: RockRockCliffEnd.L
+				Inner: RockRockCliff
+				End: RockRockCliff.L
+				Points: 2,1, 1,1, 0,1
+		MultiBrush@191: # PLATFORM
+			Template: 1008
+			Segment:
+				Start: SandPlatform.R
+				Inner: SandPlatform
+				End: SandPlatform.R
+				Points: 0,1, 1,1
+		MultiBrush@192:
+			Template: 1009
+			Segment:
+				Start: SandPlatform.R
+				Inner: SandPlatform
+				End: SandPlatform.R
+				Points: 0,1, 1,1
+		MultiBrush@193:
+			Template: 1014
+			Segment:
+				Start: SandPlatform.L
+				Inner: SandPlatform
+				End: SandPlatform.L
+				Points: 1,0, 0,0
+		MultiBrush@194:
+			Template: 1015
+			Segment:
+				Start: SandPlatform.L
+				Inner: SandPlatform
+				End: SandPlatform.L
+				Points: 1,0, 0,0
+		MultiBrush@195:
+			Template: 1005
+			Segment:
+				Start: SandPlatform.D
+				Inner: SandPlatform
+				End: SandPlatform.D
+				Points: 0,0, 0,1
+		MultiBrush@196:
+			Template: 1006
+			Segment:
+				Start: SandPlatform.D
+				Inner: SandPlatform
+				End: SandPlatform.D
+				Points: 0,0, 0,1
+		MultiBrush@196:
```

Comment:
There seems to be two `@196`s. I'm actually surprised this doesn't explode!

When I number my MultiBrushes, I typically just go with the template number, but MultiBrush names don't need to be numbers if you want to use another system. You could name them, for example, `@SandPlatformUp`.